src/test/lib/decorators_test.ts

```diff
@@ -0,0 +1,116 @@
+/**
+ * @license
+ * Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {customElement, query, queryAll} from '../../lib/decorators.js';
+import {html, LitElement} from '../../lit-element.js';
+
+const assert = chai.assert;
+
+suite('decorators', () => {
+  let container: HTMLElement;
+
+  setup(() => {
+    container = document.createElement('div');
+    container.id = 'test-container';
+    document.body.appendChild(container);
+  });
+
+  teardown(() => {
+    if (container !== undefined) {
+      container.parentElement!.removeChild(container);
+      (container as any) = undefined;
+    }
+  });
+
+  suite('@customElement', () => {
+    test('defines an element', () => {
+      @customElement('lit-element-test-1' as keyof HTMLElementTagNameMap)
```

Comment:
Use [generateElementName](https://github.com/Polymer/lit-element/blob/master/src/test/test-helpers.ts#L19) to ensure uniqueness of element name... in all tests that define elements.

Reply:
done